src/components/Header/Header.js

```diff
@@ -134,20 +134,17 @@ function Header() {
 
 function GardenNavItems({ garden }) {
   const theme = useTheme()
+  const history = useHistory()
   const token = garden.wrappableToken || garden.token
 
+  const handleOnGoToCovenant = useCallback(() => {
```

Comment:
What do you think of moving the logic to the hooks directory to DRY?

Reply:
yeah i thought about it but i think that it doesn't make too much sense to create a hook just redirect to another  page, if you feel ok with that feel free to merge it